Ray.jl/test/task.jl

```diff
@@ -149,3 +149,12 @@ end
         end
     end
 end
+
+@testset "many tasks" begin
+    n_tasks = Sys.CPU_THREADS * 2
```

Comment:
we can  maybe get away with smaller number of tasks but want to make sure we saturate teh cache of workers